iOS/Flows/Conversation/Threads/ConversationThreadCollectionViewLayout.swift

```diff
@@ -8,43 +8,29 @@
 
 import Foundation
 
-class ConversationThreadCollectionViewLayout: UICollectionViewCompositionalLayout {
+class ConversationThreadCollectionViewLayout: BottomToTopColumnCollectionViewLayout {
```

Comment:
As much as I like compositional layouts, they just don't handle bottom to top very well.